### DISCUSSION TITLE
[firebase_crashlytics] Please add Crashlytics fatal crash support

### DISCUSSION THREAD

**Discussion by kroikie** [firebase/flutterfire#5559] [Feature request]
Currently the Crashlytics plugin is limited to only report non-fatal reports. It would be nice to have the ability to report fatal reports as well.

**Comment by kroikie** [comment]
@kroikie

The issue at https://github.com/flutter/flutter/issues/29689 has been closed and moved here. Future collaboration on this issue will be done here.

**Comment by vanlooverenkoen** [comment]
This should be a high priority. In order to release production apps we should have fatal crash reports. Any progress on this?

**Comment by kroikie** [comment]
@vanlooverenkoen this is high priority for us; and we are working on it. I will leave this issue open for tracking progress.

**Comment by talhazzani** [comment]
We're counting on you @kroikie

**Comment by vanlooverenkoen** [comment]
Any update on this?

**Comment by sanekyy** [comment]
Hi there.

I see fatal crashes in Firebase Console, but logs are missed.
In non-fatal crashes logs are present.

Here is example of fatal crash. Only analytics events are present.
![image](https://user-images.githubusercontent.com/18647037/73607876-fab48180-45cc-11ea-82e8-554ea3949a9a.png)


Is it related issue?

**Comment by dccruzt** [comment]
Any update on this?

The way it is now, flutter errors are being mixed with other type of issues since we have also native code on our app. Having support for crashes will let us have more visibility of flutter code problems.

**Comment by otopba** [comment]
I have same problems =/

**Comment by Truba** [comment]
Any update on this?

I'm sorry, I know I'm just repeating the question, but I really don't want this issue to get killed due to inactivity 😊

**Comment by vanlooverenkoen** [comment]
> @vanlooverenkoen this is high priority for us; and we are working on it. I will leave this issue open for tracking progress.

@kroikie is this still a high priority? We are 1 year later. And this is still not supported?

**Comment by dotdoom** [comment]
2.5 years, in fact https://github.com/flutter/flutter/issues/14765 :-)

**Comment by rohan20** [comment]
Could we please get an ETA here? It is a bit tough to monitor a release using just non-fatal errors. This prevents us from getting an accurate number for "crash-free users" as well. Also, no spikes are reported for non-fatal errors so all monitoring needs to be done manually. 😞

**Comment by Salakar** [comment]
👋  hey all, we're currently in the process of adding a way to report a Dart error as fatal from Dart code. PR coming soon.
